contents/teams/customer-comms/objectives.mdx

```diff
@@ -1,23 +1,32 @@
-Make customer support a reliable, positive experience for paying users. 
+Make sure customers have the information they need to be successful with PostHog and deliver it in a way which sparks joy.
 
-### Hit our support SLAs for all High and Normal priority customers (Marcus + Steven)
+### Hit our expanded SLAs while keeping CSAT stable (Marcus + Steven)
 - **Rationale:** Reliable support = Good support.
-- **What we'll ship:** Scale the team to meet demand and train all new team members in support processes. 
-- **We'll know we're successful when:** We're hitting 90% SLA achievement or above for Normal and High priority tickets. 
+- **What we'll ship:** Grow the team, train other teams. 
+- **We'll know we're successful when:** 80% SLA achievement or above for _all_ Normal and High priority tickets. CSAT average does not fall below 4.1 (4.6 at start of Q3).
 
-### Drive word of mouth by improving support (Joe)
-- **Rationale:** Good support = better word-of-mouth.
-- **What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/186)
-- **We'll know we're successful when:** We're hitting our SLAs and our CSAT has either improved or remained stable. CSAT is currently 96% satisfaction. 
+### Level up our processes and automations (Steven)
+- **Rationale:** Good processes = Fewer tickets, happier users.
+- **What we'll ship:** MaxAI v2, handbook definitions (incl. definitions of what we don't support), Zendesk improvements ([domain problem](https://github.com/PostHog/meta/issues/201), team tracking, [bounced emails](https://github.com/PostHog/company-internal/issues/1324) etc).
```

Comment:
Generally in this section, I'd put MaxAI second - priority should be fixing the Zendesk bits first as they are actual problems?